backend/webapi/Features/ThirdPartyIntegrations/ThirdPartyintegrationsController.cs

```diff
@@ -22,4 +22,15 @@ public class ThirdPartyintegrationsController : PidpControllerBase
                                                                                    [FromRoute] EndorsementData.Query query)
         => await handler.HandleAsync(query)
             .ToActionResultOfT();
-}
+
+    [HttpPost("fhir/message")]
+    [Authorize(Roles = Roles.ViewEndorsements)]
```

Comment:
The ViewEndorsements role should be replaced for this endpoint. However, for version 1 of this proof of concept to allow PLR to hit our API maybe we don't need to impose a role restriction on this endpoint? We can replace this line with `[AllowAnonymous]` to specify that this endpoint does not require authorization.

Reply:
Got it. Fixed.